src/lesson34/js/contents/tab.js

```diff
@@ -100,32 +112,28 @@ const createTabContainer = () => {
     div.appendChild(tabNav.parentNode.replaceChild(div, tabNav));
 }
 
-const appendArticlesTitleFragment = (values) => {
+const createArticlesTitleFragment = values => {
```

Comment:
nit
articlesとした方がより具体的で中身が想定しやすいかと思いました
```suggestion
const createArticlesTitleFragment = articles => {
```

Reply:
@sae-github 
ありがとうございます！！
次回修正いたします✨